### PR TITLE
Enhancement jl1

### PR DESCRIPTION
**Pull request by jlandells**
Updated code to add some flexibility with connections and parameters.

Add README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Mattermost Customer Success Engineering Python Tools
+
+This repo holds any Python scripts that the Mattermost Customer Success team uses, that might also be useful to the wider community.
+
+All of these scripts will require Python, as well as the [mattermostdriver](https://vaelor.github.io/python-mattermost-driver/index.html) package.  This package can  typically be added to your environment through the user of:
```
Comment (sadohert) [inline README.md]: Would this be:

> through the user via

?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Parameter | Description
+--------- | -----------
+`-h, --help` | Displays a help message and exists
+`--siteurl SITEURL` | **REQUIRED** The URL of your Mattermost server (just the URL or IP address)
+`--port PORT` | The port on which the Mattermost servier is listening (Default: 443)
+`--scheme SCHEME` | The protocol scheme to use for communication with Mattermost.  Valid options are `http` or `https`. (Default: `https`)
+`--outfile OUTFILE` | **REQUIRED** The path of the CSV file to be created.  Note that this file must not exist, otherwise the script will exit with an error.
+`--tokenfile TOKENFILE` | **REQUIRED** The path to a file containing a [Personal Access Toekn](https://docs.mattermost.com/developer/personal-access-tokens.html) of a user with full admin privileges.  This file should contain nothing other than the access token.
+`--team TEAM` | TheThe name of the team to query. Use the url-friendly version, not the Team Display Name. If omitted, defaults to all users.
```
Comment (sadohert) [inline README.md]: > TheThe

**Review by sadohert — CHANGES_REQUESTED** [review]
Just a few minor things I noted, but otherwise lgtm... thanks for adding this @jlandells !

File: README.md (hunk 1 above)
Comment (jlandells) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (jlandells) [reply README.md]: Fixed!

**Comment by jlandells** [comment]
Changes have been pushed.  Should be ready to merge now.

**Comment by sadohert** [comment]
LGTM.  Thanks John!

**Pull request merged**